src/types.ts

```diff
@@ -75,7 +75,7 @@ export interface UserSession {
   isAccountRecoveryJourney?: boolean;
   isAccountRecoveryCodeResent?: boolean;
   accountRecoveryVerifiedMfaType?: string;
-  reauthenticate?: string;
+  reauthenticate?: string; // a truthy value represents the user in a reauthentication journey
```

Comment:
I'd probably prefer not to comment here. Perhaps it means we should change the name of the parameter to "reauthenticateEmail". The original commit message does have some explanation of this https://github.com/govuk-one-login/authentication-frontend/commit/95c4f26d2d823e9d956ed60ab420fccc91a8c751 (arguably not the best)